pam/integration-tests/cli_test.go

```diff
@@ -39,10 +40,12 @@ func TestCLIAuthenticate(t *testing.T) {
 		"Authenticate user with form mode with button":                {tape: "form_with_button"},
 		"Authenticate user with qr code":                              {tape: "qr_code"},
 		"Authenticate user and reset password while enforcing policy": {tape: "mandatory_password_reset"},
-		"Authenticate user and offer password reset":                  {tape: "optional_password_reset"},
+		"Authenticate user and reset password with mismatch":          {tape: "mandatory_password_reset_with_mismatch"},
+		"Authenticate user and offer password reset":                  {tape: "optional_password_reset_skip"},
+		"Authenticate user and accept password reset":                 {tape: "optional_password_reset_accept"},
```

Comment:
This is also covered as well, IIRC

Reply:
The reset was, but I wanted to cover this in the `login` mode when the user has to go through the button to accept the request, and we didn't cover that part.

Reply:
It's the same for CLI, but not the same in the native interface, since it implies an extra step (e.g. another input request), so I've added the test for both case again for consistency.

Reply:
Yeah, but that's not exactly the "contract" of the interactive CLI interface. I can understand trying to replicate as many tests as possible, but this is a corner case that only happens in the native model. Having it here implies that the user needs to do a specific input in order to change the password in this mode and it's not true.

Reply:
Ok, I'll drop it for the CLI.

Reply:
This wasn't removed (?)

Reply:
Oh, sorry it got confused in the various fixups... -_-

It has now.